packages/x-charts/src/ScatterChart/Scatter.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import { ScatterSeriesType } from '../models/seriesType/scatter';
+import { D3Scale } from '../hooks/useScale';
+
+export interface ScatterProps {
+  data: ScatterSeriesType['data'];
+  xDataToSvg?: D3Scale;
+  yDataToSvg?: D3Scale;
+  markerSize: number;
+}
+
+function Scatter(props: ScatterProps) {
+  const { data, xDataToSvg, yDataToSvg, markerSize } = props;
+
+  const xScale = React.useCallback((value: any) => xDataToSvg?.(value) ?? value, [xDataToSvg]);
```

Comment:
The callback is not useful here
It we want to memoize the calls to `xScale` then we should do a `useMemo` around the `data.map` (I don't know if this what we want)